src/modules/formBuilder/components/itemEditor/FormItemEditor.tsx

```diff
@@ -32,6 +32,7 @@ const FormItemEditor: React.FC<FormItemEditorProps> = ({
     // TODO: can transform server errors into RHF FieldErrors so they get inlined into the form
     // https://react-hook-form.com/ts#FieldErrors
     // errors: errorState?.errors,
+    mode: 'onChange', // allows custom RHF validator functions to run when fields change
```

Comment:
Seems fine to me, do you notice a performance impact?
> Warning: this often comes with a significant impact on performance.

Reply:
I didn't notice a performance impact, I think it still seems okay